backend/canisters/bucket/api/src/queries/file_info.rs

```diff
@@ -0,0 +1,21 @@
+use candid::CandidType;
+use serde::Deserialize;
+use types::{FileId, Hash};
+
+#[derive(CandidType, Deserialize, Debug)]
+pub struct Args {
+    pub file_id: FileId,
+}
+
+#[derive(CandidType, Deserialize, Debug)]
+pub enum Response {
+    Success(SuccessResult),
+    NotFound,
+}
+
+#[derive(CandidType, Deserialize, Debug)]
+pub struct SuccessResult {
+    pub is_owner: bool,
+    pub file_size: u64,
```

Comment:
I thought file size was a u32?

Reply:
It is in OpenChat but not OpenStorage...